tests/embedding/test_app.py

```diff
@@ -323,3 +327,111 @@ def test_app_execute_script(embedded_app):
     with pytest.raises(Exception):
         # This will throw an exception since no module named test available
         embedded_app.execute_script("import test")
+
+
+@pytest.mark.embedding
+def test_launch_ui(embedded_app, tmp_path: pytest.TempPathFactory):
```

Comment:
Is this needed since we can test launch_gui in the test_launch_gui test below?

Reply:
@klmcadams it is only needed if we use a "real" embedded app.  We could get away with a fake one here too if we mock out _is_saved.  But I think this is better as-is.